admin/database/database.go

```diff
@@ -75,3 +102,47 @@ type Project struct {
 	CreatedOn      time.Time `db:"created_on"`
 	UpdatedOn      time.Time `db:"updated_on"`
 }
+
+// User is a person registered in Rill.
+// Users may belong to multiple organizations and projects.
+type User struct {
+	ID          string
+	Email       string
+	DisplayName string    `db:"display_name"`
+	PhotoURL    string    `db:"photo_url"`
+	CreatedOn   time.Time `db:"created_on"`
+	UpdatedOn   time.Time `db:"updated_on"`
+}
+
+// UserAuthToken is a persistent API token for a user.
+type UserAuthToken struct {
+	ID           string
+	SecretHash   []byte    `db:"secret_hash"`
```

Comment:
Haha, yeah I thought about that, but this is where I landed: the reason you should have a random salt for passwords is because many users may have the same password, so without the salt, it's evident that there's a common password being used for many users (if you get the full table), and also it's easy to pre-compute hashes and check.

However, in this case, it's a hash of a uniquely generated long secret, so no one else will have the same secret. So I can't see a way that a salt increases security. Do you agree?